docs/ROADMAP.md

```diff
@@ -0,0 +1,86 @@
+# Product Roadmap
+
+MAJOR CAVEAT: This is a WORK IN PROGRESS with many items TBD. We will want weigh-in from key folks across all areas, but the idea is that this is a living, breathing document that will grow over time.
+
+### Summary
+
+There is an urgent, global need for an official WHO App to help contain and mitigate COVID-19. The app should support the WHO efforts and be available on a worldwide basis, with information tailored to the user’s location, culture and language. The app should also be sensitive to national needs and respect privacy. Once established, the system can be leveraged to proactively detect, respond early and prevent future pandemics.
+
+## Milestones
+
+Version numbering terminology:
+
+* **v0.1**: refers to the initial buildout that will be submitted to the app stores, aiming for Tuesday, March 24, 2020 ([v0.1 milestone](https://github.com/WorldHealthOrganization/app/milestone/2))
+* **v0.2+**: any versions starting with major version v0 are not intended for public distribution or consumption
+* **v1.0**: the first version that we feel offers enough value to be published onto the app stores for public download will be v1.0
+
+### v0.1
+
+**Goal:** To get a functional build of an application that can be submitted to the Apple App Store and Google Play Store for approval. This app will _NOT_ be published into those app stores for public download, but will instead serve as an approved base for future versions and to prove out that we can deliver on the technical side. The app will serve static content manually adapted from the WHO's site in English.
+
+**Target Date:** Tuesday, March 24, 2020
+
+**GitHub Milestone:** https://github.com/WorldHealthOrganization/app/milestone/2
+
+Technical Features / Requirements:
+
+* App built to be submitted on Apple App Store and Google Play Store, including code signing process
+* UI styled to exact [design spec in Figma](https://www.figma.com/file/fjzTIloCEK4FpbyDiTLj2X/iOS-UX) ([#186](https://github.com/WorldHealthOrganization/app/issues/186))
+  * Displaying static content in English only
+* App is still functional in an offline environment with default content able to be accessed without a network connection ([#120](https://github.com/WorldHealthOrganization/app/pull/120) :white_check_mark:)
```

Comment:
Remove "white check mark" as that won't update automatically. Better to see completion rate in milestone. Mousing over issue will show the status.

"Offline" should be the first word. If someone is scanning through a list, the ideal is that the first one or two words in each item will convey what's important. Conversely, if I'm scanning for "offline", it's easier to find as the first word.

```suggestion
* Offline support, so that default content works without a network connection #120
```